src/Service.Host/client/src/components/PurchaseOrders/PurchaseOrderUtility.js

```diff
@@ -346,190 +359,199 @@ function PurchaseOrderUtility({ creating }) {
     const getDateString = isoString =>
         isoString ? new Date(isoString).toLocaleDateString('en-GB') : null;
 
+    const print = () => {
+        window.print();
```

Comment:
there's probably a more react-y way to do this.. have you ever used react to print? https://www.npmjs.com/package/react-to-print

Reply:
hooks-y example here https://github.com/linn/stores/blob/master/src/Service.Host/client/src/components/tpk/Tpk.js

Reply:
I can if ya'd rather, but almost seems sort of backwards to change a line of code to several lines and add complexity with react to print. I guess accessing window isn't too react-y, but it doesn't feel entirely wrong to me (though maybe it could be in terms of react purity). I was considering getting rid of the function and just having () => window.print() in the onClick

Reply:
yeah that side of it does look quite nice, and hadn't really looked into it before either though so good to be aware of it :)